src/modules/finance/index.ts

```diff
@@ -679,9 +679,9 @@ export class FinanceModule {
    */
   currencySymbol(): string {
     let symbol: string;
-    while (!symbol) {
+    do {
       symbol = this.currency().symbol;
-    }
+    } while (!symbol);
```

Comment:
I had to look up while this is even getting looped. We should make the check more percise since the symbol itself always exists (as an empty string). Alternatively we could update the `Currency` definition to allow optional or `null` values.

```suggestion
    } while (symbol.length === 0);
```